### PR TITLE
Proposal for new pull request template

### PR DESCRIPTION
**Pull request by ryanisgrig**
Every pull request should have enough background information to contextualize for our engineers and community. Our current pull request template is too complex for contributors to properly utilize it and often components are removed on the whim of the submitter. After using it for the past few months we now have a better view of what details are relevant for a high-level understanding. I've provided a new template that I'm welcoming feedback on.

image

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
 
-### Language Understanding
-
-## Testing Steps
-<!--- Include any instructions for testing your Pull Request. Include sample utterances, steps, etc. -->
-
-## Checklist
-<!--- You can remove any items below that don't apply to the pull request. -->
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have added or updated the appropriate unit tests
```
Comment (darrenj) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: All good but I'd like to keep a nudge on testing speech if possible, important to ensure speech scenarios are thought about throughout.

**Comment by ryanisgrig** [comment]
There's a bot I'm trying to get working that can leave a comment once PRs are created. I'd like that to present the checklist to people rather than leave the mass list in the template. If I can't get it running then I'll reintroduce the list.

Sent from my??

________________________________

**Comment by darrenj** [comment]
Nice! Yes that works

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by enzocano** [comment]
> There's a bot I'm trying to get working that can leave a comment once PRs are created. I'd like that to present the checklist to people rather than leave the mass list in the template. If I can't get it running then I'll reintroduce the list.

Hi @ryanlengel, our Bot Framework SDK team developed a pipeline task ([GitHub PR Comment Task](https://marketplace.visualstudio.com/items?itemName=SOUTHWORKS.github-pr-comment)) that can add a comment on a pull-request. Are you looking for something like that?

**Review by lzc850612 — APPROVED** [review]

**Comment by ryanisgrig** [comment]
@enzocanoo I was looking at another one but it requires permissions I'm not sure I will be granted. This looks like it will accomplish the same tasks, thanks for pointing it out!

**Pull request merged**